prep-javascript-objects/main.js

```diff
@@ -0,0 +1,18 @@
+var person = {
+  firstName: 'Sophia',
+  lastName: 'Lee',
+  hobby: 'reading'
+};
+
+console.log(person);
+
+var fullName = 'Sophia' + ' ' + 'Lee';
+console.log("The person's name is: " + fullName);
```

Comment:
Hi Sophia, you can get the first name and last name from the object using dot notation.

```js
var fullName = person.firstName + ' ' + person.lastName;
```